File: redsheet-portal/src/app/database-models/v-lite-event-planning-action.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vLiteEventPlanningAction {
    constructor(vLiteEventPlanningAction?: vLiteEventPlanningAction )  {
        let properties = [ "liteEventPlanningActionBy", "liteEventPlanningActionDetail", "liteEventPlanningActionId", "liteEventPlanningActionOn", "liteMeetingManagementId",];
        for (let property of properties)
            if (vLiteEventPlanningAction && vLiteEventPlanningAction[property])
                this[property] = vLiteEventPlanningAction[property];
    }
 
    @required()
    @maxLength(50)
	liteEventPlanningActionBy : string =   undefined;
 
    @required()
    @maxLength(250)
	liteEventPlanningActionDetail : string =   undefined;
 
	liteEventPlanningActionId : number =   0 ;
 
    @required()
	liteEventPlanningActionOn : Date =   undefined;
 
    @range(1,2147483647)
	liteMeetingManagementId : number =   undefined;


}

File: RedSheet.DbEntities/Models/vLiteEventPlanningAction.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vLiteEventPlanningActions", Schema = "dbo")]
	[Serializable]
	public partial class vLiteEventPlanningAction  
	{
 
	[Required] 
	[MaxLength(50)]
	public string LiteEventPlanningActionBy { get; set; } 
 
	[Required] 
	[MaxLength(250)]
	public string LiteEventPlanningActionDetail { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int LiteEventPlanningActionId { get; set; } 
 
	[Required]
	public System.DateTime LiteEventPlanningActionOn { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int LiteMeetingManagementId { get; set; } 

	public vLiteEventPlanningAction()
	{
    }

    }
}

